docs/tailwind.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
FYI we have `docs.themeColor` options for a while to generate shades :D

Reply:
Excellent.